Babylon/groups/api/groups/connector/commands/get.py

```diff
@@ -0,0 +1,75 @@
+import json
+from logging import getLogger
+from pprint import pformat
+from typing import Optional
+
+from cosmotech_api.exceptions import NotFoundException, UnauthorizedException
+from click import argument, command, make_pass_decorator, option
+from cosmotech_api.api.connector_api import ConnectorApi
+
+from ......utils.api import (
+    convert_keys_case,
+    filter_api_response_item,
+    underscore_to_camel,
+)
+from ......utils.decorators import (
+    allow_dry_run,
+    timing_decorator,
+)
+
+logger = getLogger("Babylon")
+
+pass_connector_api = make_pass_decorator(ConnectorApi)
+
+
+@command()
+@allow_dry_run
+@timing_decorator
+@pass_connector_api
+@option(
+    "-o",
+    "--output_file",
+    "output_file",
+    help="File to which content should be outputted (json-formatted)",
+    type=str,
+)
+@argument("connector_id", type=str)
+@option(
+    "-f",
+    "--fields",
+    "fields",
+    required=False,
+    type=str,
+    help="Fields witch will be keep in response data, by default all",
+)
+def get(
+    connector_api: ConnectorApi,
+    connector_id,
+    output_file: Optional[str] = None,
+    fields: str = None,
+    dry_run: bool = False,
+):
+    """Get an registered connector data."""
+    try:
+        if dry_run:
+            logger.info("DRY RUN - Would call connector_api.find_connector_by_id")
+            retrieved_connector = {"Babylon": "<DRY RUN>"}
```

Comment:
You don't need to put the `dry_run` block in a `try` 
You can return after `dry_run` to have less indents